TitanRobotics2024/src/main/java/frc/robot/Subsystem/PositionEstimation.java

```diff
@@ -0,0 +1,80 @@
+package frc.robot.Subsystem;
+
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.estimator.DifferentialDrivePoseEstimator;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class PositionEstimation implements Subsystem
+{
+    private DriveBase driveBase;
+    private Gyro gyro;
```

Comment:
Will we even have a gyro on the competition bot?

Reply:
yes we will

Reply:
or i will be kinda annoyed